baselines/sharepointonedrive.md

```diff
@@ -0,0 +1,338 @@
+
+This section helps reduce security risks related to the sharing of files with users external to the agency. This includes guest users, users who use a verification code and users who access an Anyone link.
+
+### Policies
+#### MS.SHAREPOINT-ONEDRIVE.1.1v1
+External sharing for SharePoint SHALL be limited to Existing Guests or Only People in your Organization.
+- _Rationale:_ Sharing information outside the organization increases the risk of unauthorized disclosure. By limiting external sharing, administrators decrease the risk of unauthorized disclosure.
+- _Last modified:_ June 2023
+
+#### MS.SHAREPOINT-ONEDRIVE.1.2v1
+External sharing for OneDrive SHALL be limited to Existing Guests or Only People in your Organization.
+- _Rationale:_ Sharing files outside the organization increases the risk of unauthorized disclosure. By limiting external sharing, administrators decrease the risk of unauthorized disclosure.
+- _Last modified:_ June 2023
+
+#### MS.SHAREPOINT-ONEDRIVE.1.3v1
+External sharing SHALL be restricted to approved external domains per interagency collaboration needs.
+- _Rationale:_ By limiting sharing to domains used for interagency collaboration purposes, administrators prevent sharing with unknown organizations and individuals.
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.4v1
+Only users in approved security groups SHALL be allowed to share externally.
+- _Rationale:_ By limiting sharing to approved security groups based on interagency collaboration needs, administrators ensure only approved users can use share information outside the organization.
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+#### MS.SHAREPOINT-ONEDRIVE.1.5v1
+Guest access SHALL be limited to the email the invitation was sent to.
+- _Rationale:_ Email invitations allow external guests to access shared information. By requiring guests to sign in using the same account where the invite was sent, administrators ensure only the intended guest can use the invite.
+- _Last modified:_ June 2023
+- _Note:_ This policy is only applicable if the external sharing slider on the admin page is set to any value other than Only People in your Organization.
+
+### Resources
+
+- [Overview of external sharing in SharePoint and OneDrive in Microsoft 365 \| Microsoft Documents](https://learn.microsoft.com/en-us/sharepoint/external-sharing-overview)
+
+- [Manage sharing settings for SharePoint and OneDrive in Microsoft 365 \| Microsoft Documents](https://docs.microsoft.com/en-us/sharepoint/turn-external-sharing-on-or-off)
+
+### License Requirements
+
+- N/A
+
+### Implementation
+
+All of the settings in this section are configured in the **SharePoint admin center**.
```

Comment:
Could we change `**SharePoint admin center**` to a link to the admin center?

Reply:
Good suggestion. I'm not sure that we can though because the Sharepoint admin center link is unique to each tenant in the form of https://[TENANTNAME-admin.sharepoint.com.
Also, if we did that we would probably need to provide hyperlinks to the product-specific admin centers in all baselines to be consistent.

Reply:
FYI - for the Teams baseline, a link has been provided. Note that the tenant name is not needed in the hyperlink; it would look like: https://admin.teams.microsoft.com/ - user would be prompted to select or provide the tenant name. Agree on being consistent across products, depending on what we decide on (whether to provide a link or not) - Teams and other baselines have to be updated accordingly.

Reply:
Also, for what it's worth, it looks like in Microsoft's docs, such as https://learn.microsoft.com/en-us/sharepoint/sharepoint-admin-role, when they need a generic link to the admin center they use https://go.microsoft.com/fwlink/?linkid=2185219. This link works, just not sure how stable it will be over time.

Reply:
For GCC High the link to the Sharepoint admin center is https://[TENANTNAME-admin.sharepoint.**us**.

I tested the generic link you included above and it does not work for GCC High. See error below.
Also [see my comment about the links to Teams admin center not working in the other pull request](https://github.com/cisagov/ScubaGear/pull/397#issuecomment-1622632390).
We have the topic of including hyperlinks as a discussion to resolve at the stand-up.

![image](https://github.com/cisagov/ScubaGear/assets/107076927/4124def2-b03b-4a28-8359-5a7880c60bf2)

Reply:
I left the hyperlinks out of the baseline, based on agreement by the team at last week's stand-up meeting. The baseline simply refers to the "Sharepoint admin center".

Reply:
I'm resolving since Alden is on vacation.